test/src/test/resources/plugins/minJavaVersion-update-center.json

```diff
@@ -0,0 +1,57 @@
+updateCenter.post(
+{"connectionCheckUrl":"http://www.google.com/", // has to be here because updateDirectly sniffs only very limited formats
```

Comment:
JSON doesn't allow comments.  Does this work?

If so, why?

Reply:
> Does this work?

Yes. It works, you can see the test does use it.

> If so, why?

It's history I suppose :). This is the format used on all Jenkins update center files. 
You can check for yourself in `curl -L -r 99 https://updates.jenkins.io/update-center.json`

(BTW, a few years ago were added the actual json exports, add `.actual` in between, like https://updates.jenkins.io/update-center.actual.json).

Reply:
because it's not  JSON but JSONP, maybe comments are authorized in this format.

Reply:
AFAIK, neither JSON nor JSONP allow comment, but depending on the library you use for parsing the data, the comments are ignored.

Reply:
Would you be willing to remove the comment, or place it in an unused data element?  

I think GitHub is right to flag it with red in the diffs, since the json file format does not allow comments.

If not, I'm ok with that as well.

Reply:
I'm going to remove it. I think it works because the parser is permissive.

For the record, I didn't write this myself. There are already many similar `.json` files in that resource directory which I copied and modified for my test case.

Reply:
Done @MarkEWaite